lib/nerves_motd.ex

```diff
@@ -178,17 +178,62 @@ defmodule NervesMOTD do
     Enum.reject([days, hours, minutes, seconds], &is_nil/1)
   end
 
-  @spec clock() :: [binary(), ...]
+  @spec clock() :: binary() | [binary()]
   defp clock() do
+    local_time(NervesTimeZones) || local_time(NaiveDateTime) || utc_time()
+  end
+
+  # 2021-10-30 11:50:09-04:00 EDT America/New_York
+  defp local_time(NervesTimeZones) do
+    case Code.ensure_loaded(NervesTimeZones) do
+      {:module, mod} ->
+        mod.get_time_zone()
+        |> DateTime.now!()
+        |> DateTime.truncate(:second)
+        |> DateTime.to_string()
+
+      _ ->
+        nil
+    end
+  end
+
+  # 2021-10-30 11:49:44-0400 EDT
+  defp local_time(NaiveDateTime) do
+    if Version.match?(System.version(), "~> 1.10") do
+      case Code.ensure_loaded(NaiveDateTime) do
+        {:module, mod} ->
+          [
+            mod.local_now()
+            |> NaiveDateTime.truncate(:second)
+            |> NaiveDateTime.to_string(),
+            system_time_zone_name()
+          ]
+
+        _ ->
+          nil
+      end
+    end
```

Comment:
I really like that you're trying to support Elixir 1.9, but given that other Nerves and non-Nerves libraries are dropping support for it, I just want to drop support for it here too and save the effort of maintaining this code.

Let's just do a `NervesTimeZones` version and a Elixir 1.10+ version. I'll submit a PR to drop 1.9 to make that official.